llvm/test/Transforms/Reassociate/debugloc-factoring-neg.ll

```diff
@@ -0,0 +1,40 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S -p=reassociate < %s | FileCheck %s
+
```

Comment:
Given the name of the test, maybe we don't need it, but I wonder if throwing in a quick comment might help reduce the chance of an incorrect update_test_checks in the future?